samples/net/updatehub/overlay-prj.conf

```diff
@@ -0,0 +1,8 @@
+#Example of product_uid, you must set you product uid.
```

Comment:
It would be better that this overlay-prj.conf is called overlay-prj.conf.example, and document that user needs to create his own file and use that. This way the overlay-prj.conf is not placed into zephyr version control which could conflict user setup when he updates his tree.